octobot_trading/exchanges/exchange_websocket_factory.py

```diff
@@ -44,10 +45,19 @@ async def _create_websocket(exchange_manager, websocket_class_name, socket_manag
     try:
         exchange_manager.exchange_web_socket = socket_manager.get_websocket_client(exchange_manager.config,
                                                                                    exchange_manager)
+        if exchange_manager.exchange_backend is not None and exchange_manager.exchange.authenticated():
```

Comment:
Why here ? I don't think this is the proper place to check if an authenticated account is valid : it's the websocket client creation.

Reply:
That's why it's a WIP, so that we figure out the best way to do this check

Reply:
:+1: